AutoRest/Generators/Java/Azure.Java.Tests/src/main/java/fixtures/paging/models/ProductResult.java

```diff
@@ -1,66 +0,0 @@
-/**
```

Comment:
Oh no that is completely fine. It happens with me so many times.. Thanks for confirming.